src/EditorFeatures/CSharpTest2/Recommendations/StackAllocKeywordRecommenderTests.cs

```diff
@@ -124,12 +90,99 @@ unsafe static void Main(string[] args)
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestInsideForStatementVarDecl3()
         {
-            await VerifyAbsenceAsync(
+            await VerifyKeywordAsync(
 @"class C
 {
     unsafe static void Main(string[] args)
     {
         for (string i = $$");
```

Comment:
stackallow is legal here now??

Reply:
It is enabled for all initialization statements.

---
In reply to: [155678006](https://github.com/dotnet/roslyn/pull/23655#discussion_r155678006) [](ancestors = 155678006)

Reply:
This code is legal:
`for(Span<int> x = stackalloc int [10]; ;) {}`

---
In reply to: [155944492](https://github.com/dotnet/roslyn/pull/23655#discussion_r155944492) [](ancestors = 155944492,155678006)